backend/dal/Configuration/TenantConfigruation.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Configuration
+{
+    /// <summary>
+    /// TenantConfiguration class, provides a way to configure tenant settings in the database.
+    ///</summary>
+    public class TenantConfiguration : BaseEntityConfiguration<Tenant>
+    {
+        #region Methods
+        public override void Configure(EntityTypeBuilder<Tenant> builder)
+        {
+            builder.ToTable("Tenants");
+
+            builder.HasKey(m => m.Id);
+            builder.Property(m => m.Id).ValueGeneratedOnAdd();
+
+            builder.Property(m => m.Code).HasMaxLength(6);
```

Comment:
some codes have max length 6, others 10, others 20, do we want to try and unify this with a const?

Reply:
There is no need for consistency here outside of a decision made by the DBA.  The Tenant table isn't a lookup.  It's a separate type of entity that has a column name called Code.